compiler/resolution/virtualDispatch.cpp

```diff
@@ -462,11 +455,18 @@ static void resolveOverride(FnSymbol* pfn, FnSymbol* cfn) {
     // corresponding ones exist for the child.
     int nUserFormals = getNumUserFormals(pfn);
     for (int i = 1; i <= nUserFormals; i++) {
-      ArgSymbol* pformal = getUserFormal(pfn, i);
-      ArgSymbol* cformal = getUserFormal(cfn, i);
-      FnSymbol* pDefFn = findExistingDefaultedActualFn(pfn, pformal);
+      ArgSymbol* pFormal = getUserFormal(pfn, i);
+      ArgSymbol* cFormal = getUserFormal(cfn, i);
+      FnSymbol* pDefFn = findExistingDefaultedActualFn(pfn, pFormal);
       if (pDefFn) {
-        getOrCreateDefaultedActualFn(cfn, cformal);
+        FnSymbol* cDefFn = getOrCreateDefaultedActualFn(cfn, cFormal);
+        if (cDefFn) {
+          if (pFormal->hasFlag(FLAG_INSTANTIATED_PARAM) &&
+              cFormal->hasFlag(FLAG_INSTANTIATED_PARAM)) {
```

Comment:
In the previous check, we were also checking for type-ness of the argument. Does `FLAG_INSTANTIATED_PARAM` cover that case, too?